lesson26/src/js/module/toggle-password.js

```diff
@@ -0,0 +1,11 @@
+export const togglePassword = (passwordInput,togglePasswordButton) => {
+    if (passwordInput.type === 'password') {
+      passwordInput.type = 'text';
+      togglePasswordButton.textContent = '非表示';
+      togglePasswordButton.setAttribute('aria-label', 'パスワードを非表示');
+    } else {
+      passwordInput.type = 'password';
+      togglePasswordButton.textContent = '表示';
+      togglePasswordButton.setAttribute('aria-label','パスワードを表示');
+    }
+}
```

Comment:
The ability to show and hide passwords was made into a module because we want to use it for multiple files.